packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -2115,7 +2115,6 @@ describeCompat(
 						state: "disabled",
 					},
 				},
-				enableRuntimeIdCompressor: "on",
```

Comment:
This does NOT result in ID Compression being disabled.  The test infra enables it by default.  I lost a bunch of time trying to figure out why it was still enabled.  Better to not give the impression that this is doing something.

(same below)

Reply:
If ID compression is enabled by default, can we get rid of this option all together?

Reply:
It's off by default in production scenarios.  But the test infra opts in for this and basically every feature possible to make sure they're getting coverage.